File: SAP_APIv1/SAP_API/DTOs/Responses/Product/CreateProductResponse.cs
using System;
using System.Collections.Generic;

namespace SAP_API.DTOs.Responses
{
    public class CreateProductResponse
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public int BakingTimeInMins { get; set; }
        public int BakingTempInC { get; set; }
        public int Size { get; set; }
        public List<CreateStockedProductResponse> Stock { get; set; }

    }
}

File: bakery-app/src/models/Responses/Product/CreateProductResponse.ts
import CreateStockedProductResponse from "../StockedProduct/CreateStockedProductResponse";

export default interface CreateProductResponse {
    id: string;
    name: string;
    bakingTimeInMins: number;
    bakingTempInC: number;
    size: number;
    stock: CreateStockedProductResponse[];
}
